src/test_application/main.cpp

```diff
@@ -106,7 +108,7 @@ int main(int argc, char** argv)
 
     load_a_bunch_of_stuff();
 
-    if( ! args.value("scene").empty())
+    if(args.value("scene") != "none")
```

Comment:
`!args.value("scene").empty()` would always return true, as "simple" is set as the default value. Doing `--scene none` would just show the CLI.